### ISSUE TITLE
Script to apply database configuration options

### ISSUE THREAD

**Issue by poikilotherm** [IQSS/dataverse-kubernetes#16]
Currently, only JVM settings are configured on container startup. During the bootstrap job, a few minimal settings are set, so Dataverse is at least reachable and secured (API access).

This however is not sufficient for any further configuration. There needs to be a simple script, taking values from a `ConfigMap` and set those in the running instance. This should be separate from the bootstrapping script and idempotent, as it might/will be used frequently.

Living in the container image makes it easier to use - it is no configuration, so it should not live in a `ConfigMap` itself. It needs secrets from Kubernetes like the API unblock key and using the env var containing the correct address makes it even easier to use.

**Comment by poikilotherm** [comment]
Work started in https://github.com/IQSS/dataverse-kubernetes/tree/16-configscript

**Comment by poikilotherm** [comment]
For now, this is solved. Uncovered is automated configuration for authentication and other advanced topics. This is to be addressed in a separate issue down the road.
